deps/wazuh_testing/wazuh_testing/end_to_end/remote_operations_handler.py

```diff
@@ -0,0 +1,429 @@
+            if len(results['evidences'][evidence_key]) != len(package_data_to_use['CVE']):
+                results['checks']['all_successfull'] = False
+
+        elif operation == 'install' or operation == 'update':
+            # Check states from new package are found
+            evidence_key = 'states_not_found_to' if operation == 'update' else 'states_not_found'
+            package_data_to_use = package_data['to'] if operation == 'update' else package_data
+            results['evidences'][evidence_key] = check_vuln_state_index(host_manager, host, package_data_to_use,
+                                                                        current_datetime)
+
+            if len(results['evidences'][evidence_key]) != 0:
+                results['checks']['all_successfull'] = False
+
+
+def install_package(host: str, operation_data: Dict[str, Dict], host_manager: HostManager):
+    """
+    Install a package on the specified host.
+
+    Args:
+        host (str): The target host on which to perform the operation.
+        operation_data (dict): Dictionary containing operation details.
+        host_manager (HostManager): An instance of the HostManager class containing information about hosts.
+
+    Raises:
+        ValueError: If the specified operation is not recognized.
+    """
+    results = {
+            'evidences': {
+                "alerts_not_found": [],
+                "states_not_found": [],
+                "alerts_found": [],
+                "states_found": [],
+                "alerts_found_unexpected": [],
+                "states_found_unexpected": []
+            },
+            'checks': {}
+    }
+
+    logging.info(f"Installing package on {host}")
+
+    host_os_name = host_manager.get_host_variables(host)['os'].split('_')[0]
+    host_os_arch = host_manager.get_host_variables(host)['architecture']
+    system = host_manager.get_host_variables(host)['os_name']
+
+    if system == 'linux':
+        system = host_manager.get_host_variables(host)['os'].split('_')[0]
+
+    install_package_data = operation_data['package']
+    package_id = None
+
+    if host_os_name in install_package_data:
+        if host_os_arch in install_package_data[host_os_name]:
+            package_id = install_package_data[host_os_name][host_os_arch]
+        else:
+            raise ValueError(f"Package for {host_os_name} and {host_os_arch} not found")
+
+    package_data = load_packages_metadata()[package_id]
+    package_url = package_data['urls'][host_os_name][host_os_arch]
+
+    logging.info(f"Installing package on {host}")
+    logging.info(f"Package URL: {package_url}")
+
+    current_datetime = datetime.utcnow().isoformat()
+
+    host_manager.install_package(host, package_url, system)
+
+    logging.info(f"Package {package_url} installed on {host}")
+
+    logging.info(f"Package installed on {host}")
+
+    results['checks']['all_successfull'] = True
```

Comment:
Should the installation change the status of the result?

Reply:
Is the default value. If any of the future checks fails, final result will be False.
However, this behavior should be changed in future iterations in order to track the status of the different stages of the test